apps/examples/uart_hw_test/Kconfig

```diff
@@ -4,7 +4,8 @@
 #
 
 config UART_HW_TEST
-    bool "UART HW test application"
-    default n
-        ---help---
-	Enable UART HW test application
+	bool "UART HW test application"
```

Comment:
could you add this in another commit?

Reply:
Done

Reply:
Oh.. I meant `uart_hw_test: Fix issues in UART HW test`. Current is also good. Thank you :)